tests/snuba/rules/conditions/test_event_frequency.py

```diff
@@ -137,6 +182,47 @@ def test_batch_query_user(self):
         assert batch_query == {self.event3.group_id: 1}
 
 
+class EventFrequencyPercentConditionQueryTest(
+    EventFrequencyQueryTestBase, BaseEventFrequencyPercentTest
+):
+    rule_cls = EventFrequencyPercentCondition
+
+    @patch("sentry.rules.conditions.event_frequency.MIN_SESSIONS_TO_FIRE", 1)
+    def test_batch_query_percent(self):
+        data = {"interval": "5m", "value": 30}
+        condition_inst = self.get_rule(
+            data=data,
+            project=self.event.group.project,
+            rule=Rule(environment_id=self.environment.id),
+        )
+
+        self._make_sessions(60, self.environment2.name)
+        self._make_sessions(60, self.environment.name)
+        batch_query = condition_inst.batch_query_hook(
+            group_ids=[self.event.group_id, self.event2.group_id, self.perf_event.group_id],
+            start=self.start,
+            end=self.end,
+            environment_id=self.environment.id,
+        )
+        assert batch_query == {
+            self.event.group_id: 20,
```

Comment:
where is 20 coming from on the id here? could we reference a variable or make this a static variable?